code/dqc/timediff.q

```diff
@@ -0,0 +1,13 @@
+\d .dqc
+
+/ Takes a table name tn and two column names ca and cb, as well as a percentage
+/ pt and a tolerance tl as a timespan (eg 0D00:00:00.001).
+timediff:{[tn;ca;cb;pt;tl]
+  .lg.o[`timediff;"Checking the time differences in columns ",(", "sv string(ca;cb))," of table ",(string tn)];
+  ot:$[pt>re:1-(sum a)%count a:tl>(tn ca)-tn cb;
```

Comment:
If you switch the more than to a less than, can you get rid of the `1-` like this?

```suggestion
  ot:$[pt>re:(sum a)%count a:tl<(tn ca)-tn cb;
```

Reply:
Probably, but I've just merged it